packages/mattermost-components/package-lock.json

```diff
@@ -0,0 +1,10511 @@
+{
+  "name": "@mattermost/components",
```

Comment:
If you're not installing this separately, it shouldn't need its own package-lock.json

Reply:
Not sure what you mean? Workspaces are their own packages so they need `package.json` and `package-lock.json`

Reply:
I understand now. Played around and it does work better. Didn't know it worked like that.